pkg/uki/reset.go

```diff
@@ -55,6 +56,13 @@ func (r *ResetAction) Run() (err error) {
 		}
 	}
 
+	// SelectBootEntry sets the default boot entry to the selected entry
+	err = action.SelectBootEntry(r.cfg, "active")
+	// Should we fail? Or warn?
```

Comment:
no idea here, maybe we should only warn and not fail? After all yes, the rest is done at this point and this is a minor thing....